tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -330,6 +331,24 @@ private Task Client_ProcessEventAsync(ProcessEventArgs arg)
             return Task.CompletedTask;
         }
 
+        private bool CheckRestartAnnouncement(dynamic json) {
+            if (json is JObject) {
+                try {
+                    var messageType = (string)json["MessageType"].Value;
+                    var messageVersion = (int)json["MessageVersion"].Value;
+
+                    if (messageType == "RestartAnnouncement" && messageVersion == 1) {
+                        _restartAnnouncementReceived = true;
+                        return true;
+                    }
+                }
+                catch (Exception) {
+                    // Do nothing.
```

Comment:
>// Do nothing [](http://example.com/codeflow?start=20&length=13)

maybe a log would still be nice to have :)

Reply:
Added log.